SQL/Indentation Rules - Comments.tmPreferences

```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>scope</key>
```

Comment:
From tabs to spaces? Or spaces to tabs? (GitHub not very helpful here.)

Reply:
I changed spaces to tabs to be consistent with the other files.